scripts/jmx4perl

```diff
@@ -237,9 +236,8 @@ documentation abouting listing of MBeans.
 =cut
 
 sub command_list {
-    my ($jmx,$path) = @_;
-    $path =~ s|:|/|;
```

Comment:
I think we still have to replace the first column (indicating the end of the domain) with  a slash. The reason is, that the list operation returns a huge JSON object of this structure:

```
{
  "domain1": {
     "name=bla,type=x/y" : {
        "attr1": "val1",
        "attr2": "val2"
     },
  ...
  },
  ...
}
```

So afair you have to concatenate all part to a single path for the argument to the list opertation

```
$mbean =~ s|:|/|;
$full_path = $mbean . 
```

Reply:
Well, the replacement (colon -> slash) is done but, for consistency sake, it is done in JMX::Jmx4Perl::Request.

Reply:
yeah, sorry. I did this comment before I realized that it before in the request.